pkg/state/keys.go

```diff
@@ -75,3 +80,13 @@ func (k *txOffsetKey) bytes() []byte {
 	copy(buf[1:], k.txID)
 	return buf
 }
+
+type scoreKey struct {
+	height uint64
+}
+
+func (k *scoreKey) bytes() []byte {
+	heightBuf := make([]byte, 8)
+	binary.LittleEndian.PutUint64(heightBuf, k.height)
+	return append([]byte{scoreKeyPrefix}, heightBuf...)
```

Comment:
I think, copy is faster, better not to use append

Reply:
I can actually `PutUint64` directly into buf sub-slice, fixed.